superset/assets/src/preamble.js

```diff
@@ -29,6 +29,7 @@ if (typeof window !== 'undefined') {
   if (bootstrapData.common && bootstrapData.common.language_pack) {
     const languagePack = bootstrapData.common.language_pack;
     configure({ languagePack });
+    localStorage.setItem('locale', bootstrapData.common.locale);
```

Comment:
Does it produce the same result if you call `moment.locale()` here without using `localStorage` at all?

Reply:
You're right - I've removed the usage of `localStorage` now